pkgs/top-level/all-packages.nix

```diff
@@ -22214,6 +22214,8 @@ with pkgs;
         CoreServices;
   };
 
+  libremines = libsForQt5.callPackage ../games/libremines { };
```

Comment:
Looks like upstream supports Qt 6?

Reply:
You are correct, but should I keep both or switch to qt6?

Reply:
I have to specify `qtwayland` in `buildInputs` to make it run with wayland, otherwise `qt.qpa.plugin: Could not find the Qt platform plugin "wayland" in ""`. is this typical for qt6?

Reply:
It's time to ask @NickCao why Qt libraries aren't propagated with the hook in Qt 6...

Reply:
See https://github.com/NixOS/nixpkgs/pull/174946 for the motivation, but this feels more and more like a footgun these days....

Reply:
Whatever. I'll add it for now.